src/main/java/com/google/devtools/build/lib/remote/RemoteOptions.java

```diff
@@ -35,6 +35,17 @@
   )
   public String remoteHttpCache;
 
+  @Option(
+    name = "experimental_remote_http_cache_unix_socket",
```

Comment:
Would it be possible for this flag to also work for --remote_cache (the gRPC protocol)? gRPC also uses Netty and we already use the NettyChannelBuilder in gRPC that can be passed in a epoll and kqueue channel. The code for that is in `GoogleAuthUtils.java`.

I think we also can skip the experimental prefix. `--remote_unix_domain_socket=` would be a reasonable name? What do you think?

Reply:
I can implement support for unix sockets with gRPC but I don't have any way to test it in the real world (whereas I've tested the http via unix sockets against our work-in-progress caching setup at Stripe). I can definitely get the tests to pass, though.

For the flag name, that sounds great to me.